web/twa-vis-platform/code/src/map/mapbox/mapbox-source-utils.ts

```diff
@@ -31,16 +32,22 @@ export function addSource(map: Map, source: DataSource) {
 
     // Clone the original source definition
     const options: JsonObject = {...source.definition};
-
     // Remove properties not expected by Mapbox
     if(options["id"] != null) delete options["id"];
     if(options["metaFiles"] != null) delete options["metaFiles"];
     if(options["timeseriesFiles"] != null) delete options["timeseriesFiles"];
 
+    // Ensure data have been formatted for the app
+    if (options["data"]) {
+        const dataUrl: string = options["data"] as string;
+        options["data"] = formatAppUrl(dataUrl);
+    }
+
     // Add attributions if missing
-    if(!options["attribution"]) {
+    if (!options["attribution"]) {
         options["attribution"] = "CMCL";
     }
+    options["attribution"] = "Powered by The World Avatar | " + (options["attribution"] as string);
```

Comment:
I wonder what is best way to style this? It doesn't seem to be catching the same colour text as the default attributions